client/src/components/DragAndDrop/dNdComponents/item.jsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Grid, Typography, Paper } from '@material-ui/core';
+
+export default function ColumnItem({ columnItem, isDragging, provided, classes }) {
+    const { name, description, tag } = columnItem.content;
+    const color = tag == 0 ? '#FFF' : tag == 1 ? '#FF5D48' : tag == 2 ? '#5ACD76' : tag == 3 ? '#759CFC' : tag == 4 ? '#EDAB1D' :'#D460F7';
+    
+    return (<Paper className={isDragging ? classes.dragging : classes.columnItem}
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}>
+                <Grid container wrap="nowrap" spacing={2}>
+            <Grid item xs={12} sm={12}>
+                <Box className={classes.avatar} style={{ background: color, border: '.6px solid rgba(0,0,0,.5)'}}></Box>
```

Comment:
Ideally we don't use any inline stylings unless we absolutely have to - they are good for quick styling checks but they should be moved to a styling block/file before the feature is shipped 👍🏼